packages/components/src/circular-progress-bar/style.scss

```diff
@@ -1,5 +1,6 @@
 @import "@automattic/typography/styles/variables";
```

Comment:
If we do import from `@automattic/components`, could we at least delete this line. Afaict this file doesn't use any typography variables, and this would unnecessarily import the typography package into the dashboard code.